src/uwtools/tests/drivers/test_jedi.py

```diff
@@ -169,34 +186,14 @@ def file(path: Path):
     assert regex_logged(caplog, "Config is valid")
 
 
-def test_JEDI_configuration_file(driverobj):
-    basecfg = {"foo": "bar"}
-    base_file = Path(driverobj._driver_config["configuration_file"]["base_file"])
-    with open(base_file, "w", encoding="utf-8") as f:
-        yaml.dump(basecfg, f)
-    cfgfile = Path(driverobj._driver_config["run_dir"]) / "jedi.yaml"
-    assert not cfgfile.is_file()
-    driverobj.configuration_file()
-    assert cfgfile.is_file()
-    newcfg = YAMLConfig(config=cfgfile)
-    assert newcfg == {**basecfg, "baz": "qux"}
-
-
-def test_JEDI_configuration_file_missing_base_file(caplog, driverobj):
-    log.setLevel(logging.DEBUG)
-    base_file = Path(driverobj._driver_config["run_dir"]) / "missing"
-    driverobj._driver_config["configuration_file"]["base_file"] = base_file
-    cfgfile = Path(driverobj._driver_config["run_dir"]) / "jedi.yaml"
-    assert not cfgfile.is_file()
-    driverobj.configuration_file()
-    assert not cfgfile.is_file()
-    assert regex_logged(caplog, f"{base_file}: State: Not Ready (external asset)")
-
-
 def test_JEDI__config_fn(driverobj):
     assert driverobj._config_fn == "jedi.yaml"
 
 
+def test_JEFI__driver_name(driverobj):
```

Comment:
```suggestion
def test_JEDI__driver_name(driverobj):
```
I appreciate the irony in this one!